README.md

```diff
@@ -91,29 +91,6 @@ kermit [-h] [-v] [-d] [-c config] [-t title] [-w workdir] [-e command]
 [-e command] sets the command to execute in terminal
 ```
 
-## Key Bindings
-
-| Key                         | Action                         |
-| --------------------------- | ------------------------------ |
-| `ctrl-alt-[c]`              | copy to clipboard              |
-| `ctrl-alt-[v]`              | paste from clipboard           |
-| `ctrl-alt-[t]`              | open a new tab                 |
-| `ctrl-alt-[r]`              | reload configuration file      |
-| `ctrl-alt-[d]`              | load the default configuration |
-| `ctrl-alt-[q]`              | exit the terminal              |
-| `ctrl-alt-[k][up]`          | increase font size             |
-| `ctrl-alt-[j][down]`        | decrease font size             |
-| `ctrl-alt-[=]`              | reset font size to default     |
-| `ctrl-alt-[return]`         | open a new tab                 |
-| `ctrl-alt-[num]`            | switch to the tab number       |
-| `ctrl-alt-[l][right][pgup]` | switch to the next tab         |
-| `ctrl-alt-[h][left][pgdn]`  | switch to the previous tab     |
-| `ctrl-alt-[w][backspace]`   | close the selected tab         |
-
-• Key bindings (`ctrl-alt`) can be set to `ctrl-shift` with using the config file.
-
-• Default key bindings (`ctrl-alt`) might conflict with your desktop environments window shortcuts. To solve this issue, key bindings should be changed to `ctrl-shift`.
```

Comment:
Why this was removed? Aren't these still the default?

We should probably update the default config file to keep these default key bindings.

Reply:
No these are no longer the default. I am not sure how this should be correctly implemented. Should they be hardcoded? Should I just add them to the config in `.config/`?

Reply:
I think adding them to the `.config` as `bindi` commands would be nice.

Also, I'm okay with hardcoding them if no configuration is provided.

Reply:
I have now added the keys to the example config file.